pkg/kv/kvclient/rangefeed/rangefeed.go

```diff
@@ -165,6 +166,9 @@ func (f *Factory) New(
 // OnValue is called for each rangefeed value.
 type OnValue func(ctx context.Context, value *kvpb.RangeFeedValue)
 
+// OnValue is called for a batch of rangefeed values.
```

Comment:
nit: could you rename OnValues to OnInitialScanValues and indicate in the comment that it overrides OnValues during the initial scan? Otherwise, the client may think this callback is used outside the initial scan.

Reply:
I was thinking in the future we might want to be able to pass batches of values outside scan if a client has shown that they can consume patches, since key-by-key is just so expensive for everyone involved. Right now rangefeed doesn't have batches but if it learned to, being able to just call this for callers who already signaled their ability to consume batches seems nice.

Reply:
I added to the WithOnValues to say it is is used in the initial scan for now, but I'd like to keep the option to use it in other contexts too so I'd rather not rename it or make the comment more specific than I did